phd-tests/framework/src/test_vm/mod.rs

```diff
@@ -274,12 +269,12 @@ impl TestVm {
 
     /// Sends an instance ensure request to this VM's server, allowing it to
     /// transition into the running state.
+    #[instrument(skip_all, fields(vm = self.spec.vm_name, vm_id = %self.id))]
```

Comment:
nit, take it or leave it:  `tracing` also supports dotted paths in field name identifiers, so we *could* use `vm.id` here and everywhere else. but, this is purely stylistic, so it may not actually matter...

Reply:
We use underscores in structured field names elsewhere in the stack, so I've kept the underscores here for consistency.